config/production.json.go

```diff
@@ -0,0 +1,252 @@
+// Copyright (c) 2017 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+// Code generated by go-bindata.
+// sources:
+// config/production.json
+// DO NOT EDIT!
+
+package config
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+)
+type asset struct {
+	bytes []byte
+	info  os.FileInfo
+}
+
+type bindataFileInfo struct {
+	name    string
+	size    int64
+	mode    os.FileMode
+	modTime time.Time
+}
+
+func (fi bindataFileInfo) Name() string {
+	return fi.name
+}
+func (fi bindataFileInfo) Size() int64 {
+	return fi.size
+}
+func (fi bindataFileInfo) Mode() os.FileMode {
+	return fi.mode
+}
+func (fi bindataFileInfo) ModTime() time.Time {
+	return fi.modTime
+}
+func (fi bindataFileInfo) IsDir() bool {
+	return false
+}
+func (fi bindataFileInfo) Sys() interface{} {
+	return nil
+}
+
+var _productionJson = []byte(`{
+	"serviceName": "my-gateway",
+	"port": 4000,
+	"env": "production",
+	"useDatacenter": false,
+
+	"logger.output": "stdout",
+
+	"metrics.type": "m3",
+	"metrics.tally.flushInterval": 1000,
+	"metrics.tally.service": "my-gateway",
+
+	"metrics.m3.hostPort": "127.0.0.1:9052",
+	"metrics.m3.service": "my-gateway",
+	"metrics.m3.flushInterval": 500,
+
+	"metrics.runtime.enableCPUMetrics": true,
+	"metrics.runtime.enableMemMetrics": true,
+	"metrics.runtime.enableGCMetrics": true,
+	"metrics.runtime.collectInterval": 1000,
+
+	"tchannel.serviceName": "my-gateway",
+	"tchannel.processName": "my-gateway"
+}
+`)
+
+func productionJsonBytes() ([]byte, error) {
```

Comment:
It's a generated function. It returns the bytes for productionJson, and it's a private function